### PR TITLE
Updates Staking Tab names

### LINKED ISSUE #3218: Staking Tab - "History Tickets" and "Tickets Status" name change
Currently the tab to view Ticket History is named "History Tickets", which does not sound like proper English.

image

I believe it would be more appropriate to rename these tabs to "Ticket Status" and "Ticket History".

### PR DESCRIPTION
**Pull request by dezryth**
Closes #3218 

Updates Tickets Status to Ticket Status.
Updates History Tickets to Ticket History.

image

### CODE REVIEW

File: app/i18n/translations/original.json [hunk 1]
```diff
@@ -1094,10 +1094,10 @@
   "tickets.startAutoBuyerConfirmation.description": "Decrediton must remain running for tickets to be automatically purchased.",
   "tickets.startAutoBuyerHeader": "Ticket buyer is now running.",
   "tickets.stopAutoBuyerHeader": "Ticket buyer is now stopped.",
-  "tickets.tab.mytickets": "History Tickets",
+  "tickets.tab.mytickets": "Tickets History",
```
Comment (davecgh) [inline app/i18n/translations/original.json:1097]: Ticket

**Review by davecgh — CHANGES_REQUESTED** [review]

**Comment by amassarwi** [comment]
@dezryth Thanks for bringing this to the table.

My thoughts from matrix chat room:

These indeed read better than current labels, just thought as we already changed this page title and it's menu item label
 from `Tickets` to `Staking` lately maybe we should use it also in tabs labels.

Something like: `Stake`, `Staking Status`, `Staking History` & `Statistics` ?

**Comment by dezryth** [comment]
@amassarwi I personally feel inclusion of the word "Ticket" in most of these tab names is the most appropriate as the tabs display information on ticket purchasing and tickets purchased by the wallet, while "Stake/Staking" will lead to a little more mental gymnastics to understand what the context is. Perhaps more importantly, since Tickets are a huge differentiator for Decred as opposed to other cryptocurrencies that have PoS as part of their consensus and "Staking", I think it's important for us to keep the term "Ticket" around somewhat prominently. This is my opinion, but interested to hear what others think. I don't feel too strongly about it one way or the other.

File: app/i18n/translations/original.json (hunk 1 above)
Comment (dezryth) [reply app/i18n/translations/original.json]: Fixed. Thanks! Not sure how I missed this one.

**Comment by amassarwi** [comment]
I'm fine with both, was just a suggestion.

**Comment by amassarwi** [comment]
Please mention the issue linked to this PR in the description above.

See github [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for linking issues with PRs - after doing so, merging the PR successfully will automatically close the
linked issue.

**Review by davecgh — APPROVED** [review]

**Review by alexlyp — APPROVED** [review]

**Pull request merged**